scripts/build.js

```diff
@@ -24,25 +29,118 @@ const walk = function(dir) {
   return results
 }
 
-const allFiles = walk('./src/components')
-const allTwigFiles = allFiles.filter(file => file.substr(-5, 5) === '.twig')
-allTwigFiles.forEach(file => {
-  fs.copy(file, file.replace('./src/', './build/'))
-})
-
+// Compile dependencies
 const compiler = webpack(webpackConfig)
 compiler.run((err, stats) => {
   const report = stats.toJson()
   if (report.warnings.length) {
     report.warnings.forEach(message => console.warn(message))
   }
   if (report.errors.length) {
-    report.errors.forEach(message => console.error(message))
+    report.errors.forEach(message => {
+      throw new Error(message)
+    })
   } else if (err) {
-    console.error(err)
+    throw err
   } else {
-    console.timeEnd('Build completed in')
-    console.log(stats.hash, 'Done!')
+    createDistributionPackage(stats.hash).then(distStats => {
+      console.log(stats.hash, 'Done!')
+      createReport(distStats)
+    })
   }
 })
+
+const createReport = ({ hash, filename, size }) => {
+  const date = new Date()
+  const stats = {
+    hash,
+    date,
+    filename,
+    size
+  }
+  fs.writeFileSync(`${distFolder}/feed.json`, JSON.stringify(stats))
+
+  fs.writeFileSync(
+    `${distFolder}/index.html`,
+    `<!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Distribution ${date}</title>
+  </head>
+  <body>
+    <div class="stats">
+${Object.keys(stats)
+  .map(stat => {
+    return `      <div class="stat stat-${stat}">
+        <label style="font-weight:bold;">${stat}:</label>
+        ${stats[stat]}
+      </div>
+`
+  })
+  .join('')}    </div>
+    <a href="${filename}" title="download ${filename}" download>Download</a>
+  </body>
+  </html>`
+  )
+
+  console.timeEnd('Build completed in')
+}
+
+const createDistributionPackage = hash =>
+  new Promise((resolve, reject) => {
+    console.time('Distribution bundle created in')
+    const filename = `${hash}.zip`
+    const output = fs.createWriteStream(`${distFolder}/${filename}`)
+    const archive = archiver('zip', { zlib: { level: 9 } })
+
+    output.on('finish', function() {
+      console.timeEnd('Distribution bundle created in')
+      console.log(archive.pointer() + ' total bytes')
+
+      resolve({
+        hash,
+        filename,
+        size: archive.pointer()
+      })
+    })
+
+    archive.on('warning', function(err) {
+      if (err.code === 'ENOENT') {
+        console.warn(err.message)
+      } else {
+        // throw error
+        reject(err)
+      }
+    })
+
+    archive.on('error', function(err) {
+      reject(err)
+    })
+
+    archive.pipe(output)
+
+    // Add Components to Archive
+    const allFiles = walk('./src/components')
```

Comment:
U can change this into  const allFiles = walk('./src') to retrieve all the files